rc.conf

```diff
@@ -45,3 +45,8 @@
 # RNG, for example if you plan to replicate this file system image and do not
 # have the wherewithal to first delete the contents of /var/lib/seedrng.
 #SEEDRNG_SKIP_CREDIT=false
+
+# The device manager to run during the init process. This is killed once boot
+# is complete for a supervised version to take its place. Valid values are
+# currently 'mdevd' or 'udevd' (default).
+CONFIG_DEV=udevd
```

Comment:
I'd change the variable name to `CONFIG_DEVD`, or something like that, as I think the `_DEV` suffix would imply a *device*. But that's bikeshedding, feel free to ignore.